tools/test/audio/comp_run.sh

```diff
@@ -16,7 +16,7 @@ HOST_ROOT=../../testbench/build_testbench
 HOST_EXE=$HOST_ROOT/install/bin/testbench
 HOST_LIB=$HOST_ROOT/sof_ep/install/lib
 TPLG_LIB=$HOST_ROOT/sof_parser/install/lib
-TPLG_DIR=../../test/topology
+TPLG_DIR=../../build_tools/test/topology
```

Comment:
Oh, I see now that I've been generating the topologies into wrong directory. Thanks!